kafka/generated_errors.go

```diff
@@ -1,6 +1,6 @@
 package kafka
 // Copyright 2016 Confluent Inc.
-// AUTOMATICALLY GENERATED BY /Users/magnus/gocode/bin/go_rdkafka_generr ON 2019-07-15 15:16:46.48088 +0200 CEST m=+0.000708503 USING librdkafka 1.1.0
+// AUTOMATICALLY GENERATED BY /home/rigelbm/go/bin/go_rdkafka_generr ON 2019-12-16 13:32:50.652785506 +0000 GMT m=+0.000071676 USING librdkafka 1.2.2-RC1-35-g4ffe54
```

Comment:
Looks like you did not generate this with librdkafka v1.3.0, or your librdkafka was not a tagged build (`git fetch --tags && ./configure --reconfigure && make clean && make -j && sudo make install`)